### PR TITLE
Microverse Coding Challenge review

### PR DESCRIPTION
**Pull request by Dadadon**

### CODE REVIEW

**Comment by Dadadon** [comment]
Great job so far on the project just a few minor changes left. Your readme file is not descriptive. Please add descriptive README file to your project - please use this [template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Happy coding

**Comment by wuletawwonte** [comment]
## Hi @Dadadon 👋🏿👋🏿👋🏿, 

Good job on the review so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Clear and an unambiguous review comment.
✔️ Great suggestions.

## Required Changes ♻️

- [ ] You have written only one comment. Please write at least two comments in your review.  
- [ ] Please use the "Review changes" feature from GitHub to review your pull request. Please refer [this lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) to read the lesson. 
- [ ] Kindly use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments.

## Optional suggestions

- [ ] [OPTIONAL] Please use a couple of emojis and/or gifs in your review. They make the code review more human-friendly 👽🤖.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dadadon — COMMENTED** [review]
Here is a suggestion you can add a live version link of your page to the README.md file for your professional project using [github pages](https://pages.github.com/). Here are some [detailed instructions](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site) on how to do so.

Happy coding!

File: README.md [hunk 1]
```diff
@@ -4,4 +4,4 @@
 
 You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it
```
Comment (Dadadon) [inline README.md:6]: Here is a suggestion you can add a live version link of your page to the README.md file for your professional project using [github pages](https://pages.github.com/). Here are some [detailed instructions](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site) on how to do so.

Happy coding!

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Dadadon, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Aspects to improve ♻️

- [ ] Kindly use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments.
- [ ] The project requirements specify that the student should write at least two comments. Try to start from the beginning and write 2 inline comments of issues you found in the code and give a proper review again.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -4,4 +4,4 @@
...
 
-I would like to thank Muhammad Sesay for allowing to use his code for the review.
\ No newline at end of file
+I would like to thank Muhammad Sesay for allowing me to use his code for the review.
```
Comment (Dadadon) [inline README.md:7]: Great job so far on the project just a few minor changes left. Your readme file is not descriptive. Please add descriptive README file to your project - please use this [template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Happy coding

**Comment by Nemwel-Boniface** [comment]
## Hello @Dadadon  :wave: 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job reviewing your work so far! You have done tremendous work identifying that there are some areas that need to be worked on. 

However, as per requirement, you should've made the comments referencing the code using Github's review function. Here is a refresher [video](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d) on how to make use of GitHub inline comment features. As of now, it would be hard for even you to know exactly where to make the change.

## Highlights

 :heavy_check_mark:  Comments are polite suggestions rather than orders.
 :heavy_check_mark:  Good explanation as to why the changes are requested.
 :heavy_check_mark:  Clear and easy to understand comments.

## ♻️ Required Changes ♻️

Please follow the guidelines and make the comments using Github's review function.

- Go to your [Files changed](https://github.com/Dadadon/Microverse-Challenge/pull/1/files) section of the PR.
- Find the piece of code interested in the requested changes.
- Add a new comment referencing that line of code using the + icon next to the line.

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Dadadon** [comment]
I did that, I did all that was asked of me  and I still have to submit a 4th review?
    On Friday, July 8, 2022 at 02:22:46 PM EST, Nemwel Boniface ***@***.***> wrote:  
 
 


Hello @Dadadon 👋
 
I'm Nemwel your reviewer for this PR 🧔‍♂️
 
Good job reviewing your work so far! You have done tremendous work identifying that there are some areas that need to be worked on.

However, as per requirement, you should've made the comments referencing the code using Github's review function. Here is a refresher video on how to make use of GitHub inline comment features. As of now, it would be hard for even you to know exactly where to make the change.

Highlights

 ✔️ Comments are polite suggestions rather than orders.
 ✔️ Good explanation as to why the changes are requested.
 ✔️ Clear and easy to understand comments.

 ♻️ Required Changes ♻️
 
Please follow the guidelines and make the comments using Github's review function.
   
   - Go to your Files changed section of the PR.
   - Find the piece of code interested in the requested changes.
   - Add a new comment referencing that line of code using the + icon next to the line.

[Optional] suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 🥳 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Nemwel-Boniface in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Review by bimbolabuari — COMMENTED** [review]
Hello @Dadadon 👋,

_This is an additional review as a result of the second opinion request._

- As requested by the previous code reviewers, Kindly make inline review comments referencing the code using Github's review function. Here is a refresher on how to make use of GitHub inline comment features

  - Go to your [Files changed](https://github.com/Dadadon/Microverse-Challenge/pull/1/files) section of the PR.
  - Find two pieces of code you want to request changes on.
  - Add a new comment referencing that line of code using the + icon next to the line.
  - After writing the review, click on the `review changes` button to add a summary comment and submit your review. 

Keep rocking 🚀!

File: README.md [hunk 3]
```diff
@@ -4,4 +4,4 @@
```
Comment (Dadadon) [inline README.md:4]: Please, kindly update this section of your readme by adding the following headers:

1. Contribution header: Kindly add all the contents in the contributing header, as you've removed some contents from it. You can refer to the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) to add the missing contents.
2. Show your support header.
 
The arrangement should follow that used in the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) to keep consistency to make your repo appear professional.

File: README.md (hunk 1 above)
Comment (Dadadon) [inline README.md:6]: Please, kindly add the Getting started headers by telling the user how to have your project running locally on their machine. Example:
### **Getting started**
To get a local copy up and running follow these simple example steps.

- How to clone the repo, how to move into the cloned repo,
- Prerequisite: e.g vscode editor, a web browser(chrome, etc).

**Review by Dadadon — COMMENTED** [review]
Great job so far! Just a few minor changes left

**Review by ishimwezachee — APPROVED** [review]
Hi @Dadadon , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
You've implemented the changes from previous reviewers 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Nemwel-Boniface) [reply README.md:6]: Hello @Dadadon :wave:
You have done well making the changes that were requested. kindly always take the requested changes positively as we are here to help you a world class developer.

Cheers and all the best #happycoding :rocket:

**Comment by Dadadon** [comment]
I honestly didn't know what I was doing wrong. I figured it out I made the requested changes but I didn't submit the changes so they were all on pending. So I could see them but you couldn't 
    On Saturday, July 9, 2022 at 01:14:18 PM EST, Nemwel Boniface ***@***.***> wrote:  
 
 


@Nemwel-Boniface commented on this pull request.

In README.md:
> @@ -4,4 +4,4 @@
 
 You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it
 

Hello @Dadadon 👋
You have done well making the changes that were requested. kindly always take the requested changes positively as we are here to help you a world class developer.

Cheers and all the best #happycoding 🚀

**Pull request merged**